wallet/src/shared/cryptography/mnemonics.ts

```diff
@@ -22,12 +22,16 @@ export function generateMnemonic(): string {
 }
 
 /**
- * Derive public key and private key from the Mnemonics
+ * Get public key and private key from the Mnemonics
  * @param mnemonics a 12-word seed phrase
  * @returns public key and private key
  */
 export function getKeypairFromMnemonics(mnemonics: string): Ed25519KeypairData {
```

Comment:
we can try deprecate this method if the wallet team approves. we should always derive the key using SLIP-0010 (maybe with the default path to begin with)

Reply:
We're trying to move to a multi-account setup in the wallet in the future, so I'd be in favor of deprecating this method to start preparing for that.